codegenerator/cli/src/hbs_templating/codegen_templates.rs

```diff
@@ -477,18 +476,38 @@ impl ContractTemplate {
             .map(|event| EventTemplate::from_config_event(event))
             .collect::<Result<_>>()?;
 
-        let event_signatures = contract
-            .events
-            .iter()
-            .map(|event| event.get_event_signature())
-            .collect();
+        let module_code = match &contract.abi {
+            Abi::Evm(abi) => {
+                let signatures = abi.get_event_signatures();
+
+                format!(
+                    "let abi = Ethers.makeAbi((%raw(`{}`): Js.Json.t))\nlet eventSignatures = [{}]",
```

Comment:
I don't really mind too much but sometimes these multiline template strings read easier with a rust raw string. Kind of like template strings in js. https://rahul-thakoor.github.io/rust-raw-string-literals/

Reply:
I'll change in the next PR